docs/guides/automations.md

```diff
@@ -0,0 +1,347 @@
+
+!!! Tip "Find deployment_id from the CLI"
+      The quickest way to see the ID's associated with your deployment would be running `prefect deployment ls`
+      in an authenticated command prompt, and you will be able to see the id's associated with all of your deployments
+
+```bash 
+prefect deployment ls
+                                          Deployments                                           
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ Name                                                  ┃ ID                                   ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│ Extract islands/island-schedule                       │ d9d7289c-7a41-436d-8313-80a044e61532 │
+│ build-names/deploy-build-names                        │ 8b10a65e-89ef-4c19-9065-eec5c50242f4 │
+│ ride-duration-prediction-backfill/backfill-deployment │ 76dc6581-1773-45c5-a291-7f864d064c57 │
+└───────────────────────────────────────────────────────┴──────────────────────────────────────┘
+``` 
+We can create an automation via a POST call, where we can programatically create the automation. Ensure you have your `api_key`, `account_id`, and `workspace_id` are handy. 
+
+```python
+def create_event_driven_automation():
+    api_url = f"https://api.prefect.cloud/api/accounts/{account_id}/workspaces/{workspace_id}/automations/"
+    data = {
+    "name": "Event Driven Redeploy",
+    "description": "Programmatically created an automation to redeploy a flow based on an event",
+    "enabled": "true",
+    "trigger": {
+    "after": [
+        "string"
+    ],
+    "expect": [
+        "prefect.flow-run.Running"
+    ],
+    "for_each": [
+        "prefect.resource.id"
+    ],
+    "posture": "Proactive",
+    "threshold": 30,
+    "within": 0
+    },
+    "actions": [
+    {
+        "type": "run-deployment",
+        "source": "selected",
+        "deployment_id": "YOUR-DEPLOYMENT-ID", 
+        "parameters": "10"
+    }
+    ],
+    "owner_resource": "string"
+        }
+    
+    headers = {"Authorization": f"Bearer {PREFECT_API_KEY}"}
+    response = requests.post(api_url, headers=headers, json=data)
+    
+    print(response.json())
+    return response.json()
+```
+ 
+After running this function, you will see within the UI the changes that came from the post request. Keep in mind, the context will be "custom" on UI. 
+
+Let us run the underlying flow and see the deployment get kicked off after 30 seconds elapsed. This will result in a new flow run of `build_names`, and we are able to see this new deployment get initiated with the custom parameters we outlined above. 
+
+In a few quick changes, we are able to programatically create an automation that deploys workflows with custom parameters. 
+
+## Using an underlying .yaml file
```

Comment:
i think both the python script and this should just be scrapped for using the triggers key on prefect.yaml